src/Form/index.tsx

```diff
@@ -617,6 +584,9 @@ function Form({
       updateFieldOptions
     };
 
+    // Clear auth (and other) loaders and reset auth flag
+    redirectAfterLoginRef.current = false;
```

Comment:
couldn't this be set in getNextAuthStep instead? so this logic is in the hook

Reply:
sent on slack. posting here for posterity: 

yeah I wanted to move where the flag is reset to false, but earlier when I tried colocating with the other auth code it wasn’t working properly. It looks like with these other changes I can reset it in the auth hook

but I still need to return the ref rather than the value because otherwise the value is stale (stuck at true for the last render), which then prevents navigation to any other step because the code sees the flag as true so it sets the next step to the login start step